dotenv/README.md

```diff
@@ -9,127 +9,131 @@ Setup a `.env` file in the root of your project.
 
 ```ts
-// app.ts
-import { config } from "https://deno.land/std@$STD_VERSION/dotenv/mod.ts";
+import { load } from "https://deno.land/std@$STD_VERSION/dotenv/mod.ts";
 
-console.log(await config());
+await load();
+console.log("GREETING", Deno.env.get("GREETING"));
 ```
 
 Then run your app.
 
 ```
-> deno run --allow-read app.ts
-{ GREETING: "hello world" }
+> deno run --allow-read --allow-env app.ts
+GREETING hello world
 ```
 
-### Options
+## Load
 
-- `path?: string`: Optional path to `.env` file. Defaults to `./.env`.
-- `export?: boolean`: Set to `true` to export all `.env` variables to the
-  current processes environment. Variables are then accessable via
-  `Deno.env.get(<key>)`. Defaults to `false`.
-- `safe?: boolean`: Set to `true` to ensure that all necessary environment
-  variables are defined after reading from `.env`. It will read `.env.example`
-  to get the list of needed variables.
-- `example?: string`: Optional path to `.env.example` file. Defaults to
-  `./.env.example`.
-- `allowEmptyValues?: boolean`: Set to `true` to allow required env variables to
-  be empty. Otherwise it will throw an error if any variable is empty. Defaults
-  to `false`.
-- `defaults?: string`: Optional path to `.env.defaults` file which defaults to
-  `./.env.defaults`.
+```ts
+import { load } from "https://deno.land/std@$STD_VERSION/dotenv/mod.ts";
+
+const object = await load(Deno.env, { envPath: "path/to/.env" });
+```
 
-### Auto loading
+## LoadSync
 
-`load.ts` automatically loads the local `.env` file on import and exports it to
-the process environment:
+```ts
+import { loadSync } from "https://deno.land/std@$STD_VERSION/dotenv/mod.ts";
+
+const object = loadSync(Deno.env, { envPath: "path/to/.env" });
+```
+
+### Load options
+
+- `envPath?: string | URL`: the path or url to the .env file. Defaults to
+  ".env".
+- `examplePath?: string | URL`: the path or url to the .env.example file.
+  Defaults to ".env.example".
+- `defaultsPath?: string | URL`: the path or url to the .env.defaults file.
+  Defaults to ".env.defaults".
+- `allowEmptyValues?: boolean`: set to `true` to allow variables with an empty
+  value.
+
+## Auto loading
+
+`load.ts` automatically loads the local `.env` file, verifies it against
+`.env.exanple` if existent, adds defaults from `.env.defaults` if existent and
+exports it to the process environment object:
 
 ```sh
 # .env
 GREETING=hello world
 ```
 
 ```ts
-// app.ts
 import "https://deno.land/std@$STD_VERSION/dotenv/load.ts";
 
-console.log(Deno.env.get("GREETING"));
+console.log(Deno.env.get("GREETING")); // hello world
 ```
 
-```
-> deno run --allow-env --allow-read app.ts
-hello world
-```
+## Parsing Rules
 
-### Safe Mode
+The parsing engine currently supports the following rules:
 
-To enable safe mode, create a `.env.example` file in the root of the project.
+- `BASIC=basic` becomes `{BASIC: 'basic'}`
+- empty lines are skipped
+- lines beginning with `#` are treated as comments
```

Comment:
Maybe rephrase that anything after `#` is treated as a comment (ie. `FOO=bar # this is a comment` works)